docs/source/developers/guide/index.rst

```diff
@@ -134,7 +134,8 @@ of adding a basic feature.
 
 #. **Once you implemented the planned fix or feature, write and run tests for it**
 
-   See detailed instructions on how to :ref:`test <testing>`.
+   See detailed instructions on how to :ref:`test <testing>`. Also check the
+   :ref:`syle of the code<styling>` before proceeding to the next step!
```

Comment:
It's a little unclear what it means to check the style.  Could we change this to a more direct instruction, e.g. "run the styler" or "run the linter to make sure the code is styled correctly", or something like that?

Reply:
Yeah, that would be better! Will do.